api/app/GraphQL/Validators/DepartmentSpecificRecruitmentProcessPositionBelongsToMany.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\GraphQL\Validators;
+
+use Nuwave\Lighthouse\Validation\Validator;
+
+final class DepartmentSpecificRecruitmentProcessPositionBelongsToMany extends Validator
+{
+    /**
+     * Return the validation rules.
+     *
+     * @return array<string, array<mixed>>
+     */
+    public function rules(): array
+    {
+        return [
+            'create' => ['array', 'min:1'],
```

Comment:
I just have a question about this validation. Right now, if I simply omit the `position` key from variables this rule passes. Is that intended or is position a required field that must have at least one selected?

Reply:
Okay, thanks! Just making sure, based off the issue I wasn't 100% sure if it was required or not 😅